_getting-started/index.md

```diff
@@ -7,55 +7,117 @@ includeTOC: true
 
 redirect_from: "/getting-started.html"
 ---
+## Try Scala without installing anything 
+You can try Scala, with access to all Scala compilers and 
+all published libraries with [Scastie](https://scastie.scala-lang.org/)
```

Comment:
It might be nicer to point to a hello-world kind of script rather than a blank page. Just like you're importing the hello world template with `sbt new` below, I believe this makes a better intro. Starting with a blank page, a complete beginner wouldn't even know what to write down.

Or instead, suggest that the user can type in
```scala
println("Hello world!")
```
for example. At least they get *something* working.